contributors/distribution-guidelines.md

```diff
@@ -1,17 +1,42 @@
 # Guidelines for Distributions
 
-So you want to distribute your own version of a Chef product:
+So you want to create a new distro of a Chef product….
 
-Congratulations! We welcome distributions of any and all of our products with open arms.
+Excellent! Welcome aboard!
 
-We do require that distributions follow our trademark policy https://www.chef.io/trademark-policy/
+Anyone is welcome to make a distribution of Chef’s Open Source products, but must remove Chef trademarks from the code base (the full Chef trademark policy can be found here). 
```

Comment:
Looks like "can be found here" was intended to be a link. I offer this slight rewording to avoiding using "here" links in hypertext documents.

```suggestion
Anyone is welcome to make a distribution of Chef’s Open Source products, but must remove Chef trademarks from the code base (see the full [Chef trademark policy](https://www.chef.io/trademark-policy/) for details). 
```